### ISSUE TITLE
Update Croatian translations

### ISSUE THREAD

**Issue by vedranmiletic** [squidfunk/mkdocs-material#4936]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "hr",
  "action.edit": "Uredi stranicu",
  "action.skip": "Preskoči na sadržaj",
  "action.view": "Pregledaj izvorni kod ove stranice ⬅️",
  "announce.dismiss": "Ne prikazuj ovo opet",
  "blog.archive": "Arhiva ⬅️",
  "blog.categories": "Kategorije ⬅️",
  "blog.categories.in": "u ⬅️",
  "blog.continue": "Nastavi čitati ⬅️",
  "blog.draft": "Nacrt ⬅️",
  "blog.index": "Natrag na indeks ⬅️",
  "blog.meta": "Metapodaci ⬅️",
  "blog.references": "Srodne poveznice ⬅️",
  "clipboard.copy": "Kopiraj u međuspremnik",
  "clipboard.copied": "Kopirano u međuspremnik",
  "consent.accept": "Prihvati",
  "consent.manage": "Upravljaj postavkama",
  "consent.reject": "Odbij",
  "footer": "Podnožje",
  "footer.next": "Sljedeće",
  "footer.previous": "Prethodno",
  "header": "Zaglavlje",
  "meta.comments": "Komentari",
  "meta.source": "Izvor",
  "nav": "Navigacija",
  "readtime.one": "1 minuta čitanja ⬅️",
  "readtime.other": "# minut(a/e) čitanja ⬅️",
  "rss.created": "RSS izvor ⬅️",
  "rss.updated": "RSS izvor osvježenog sadržaja ⬅️",
  "search": "Pretraživanje",
  "search.placeholder": "Pretraži",
  "search.share": "Podijeli",
  "search.reset": "Očisti",
  "search.result.initializer": "Inicijaliziranje pretraživanja",
  "search.result.placeholder": "Unesite pojam pretraživanja",
  "search.result.none": "Ništa nije pronađeno",
  "search.result.one": "1 rezultat pretraživanja",
  "search.result.other": "# rezultat(a) pretraživanja",
  "search.result.more.one": "1 rezultat na ovoj stranici",
  "search.result.more.other": "# rezultat(a) na ovoj stranici",
  "search.result.term.missing": "Nedostaje",
  "select.language": "Odabir jezika",
  "select.version": "Odabir verzije",
  "source": "Idi u repozitorij",
  "source.file.contributors": "Suradnici ⬅️",
  "source.file.date.created": "Stvaranje",
  "source.file.date.updated": "Posljednje ažuriranje",
  "tabs": "Kartice",
  "toc": "Sadržaj",
  "top": "Vrati se na vrh"
}[key] }}{% endmacro %}
```

**Comment by vedranmiletic** [comment]
I added the missing translations and improved some of the existing ones.

**Comment by squidfunk** [comment]
Great, thanks! At some point we need to switch to `gettext` to cater to the cases where you needed to use (a/e), so we can provide more accurate translations, but for now this should be fine.

**Comment by squidfunk** [comment]
Added in 664af42367c696abf0d7f58e3903fb44579cde89.

**Comment by squidfunk** [comment]
Released as part of [9.0.8](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.8).
